iceddev/git-splainin
# initial implementation of styling - added primed testbed

#### What's this PR do?

Adds a fancy UI to the options page.  Does a bit of refactoring of the react components into modules, etc.

I also started working on `primed` but just inside this repo.  I will pull it out into a module when I feel it is good enough.
#### Where should the reviewer start?

Fetch and rebuild.
#### How should this be manually tested?

Check out the options page.  Make sure all the old stuff still works and you can navigate with the menu.  The template textarea isn't wired up to anything, the cancel button isn't wired up.  If the options page is refreshed, the user is returned to the "template" tab.
#### Any background context you want to provide?

Figured we needed some cool styling to be legit.
#### What are the relevant tickets?

None
#### Screenshots (if appropriate)

![screen shot 2015-05-19 at 7 20 41 pm](https://cloud.githubusercontent.com/assets/992373/7718071/49b963c2-fe60-11e4-80b4-5b2de12eb744.png)
![screen shot 2015-05-19 at 7 34 47 pm](https://cloud.githubusercontent.com/assets/992373/7718070/49b75212-fe60-11e4-835b-22fe39b7d323.png)
